[Linting] Fixed useExhaustiveDependencies in biome (#3197)
navikt/Designsystemet: .changeset/bright-meals-repair.md

---
"@navikt/ds-react": patch
---

> Performance: :zap: Optimized memoization for rerendring in some components.

Reviewer: Maybe we could mention the affected components, since it's only 3 afaics.

> Performance: :zap: Optimized memoization for rerendring in some components.

Author: ```suggestion
Performance: :zap: Optimized memoization for rerendring in Combobox and Modal.
```

Reviewer: tbh I suspect these changes are barely measurable when it comes to performance, but we can use your suggested text if you just need a changelog entry "for the record" since you touched some components.

Author: Yea i doubt these have any affect at all
